### DISCUSSION TITLE
ubuntu24.04

### DISCUSSION THREAD

**Discussion by Iannnnnnnnnnnnnnnnn** [Icarusradio/intel-media-driver-dg1#8] [Q&A]
请问ubuntu 24.04的驱动是按他说的https://github.com/intel-gpu/intel-gpu-i915-backports/issues/168 等下一次rolling版本就可以用了吗

**Comment by Icarusradio** [comment]
应该是的，不过我没试过。
Ubuntu 24.04 用的是 6.8 内核，自带 xe 内核模块。我试过 xe 是可以直接驱动 DG1 的，不需要打补丁。但是 Jellyfin 目前不支持使用 xe 内核驱动转码，所以还是得给 i915 内核驱动打补丁使用。
